.pre-commit-config.yaml

```diff
@@ -1,15 +1,16 @@
 repos:
 -   repo: https://github.com/pycqa/isort
-    rev: 5.7.0
+    rev: 5.10.1
     hooks:
     -   id: isort
 -   repo: https://github.com/ambv/black
-    rev: 22.3.0
+    rev: 22.8.0
     hooks:
     -   id: black
         language_version: python3
+        exclude: elasticapm\/utils\/wrapt
 -   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.4
+    rev: 3.9.2
```

Comment:
why not latest? 5.0.4

And we should target github, not their mirror on gitlab.

Reply:
good catch, updated in 7823421b

Reply:
to avoid have to chekc for new version you can add `.github/dependabot.yaml` file like this one

```yaml
version: 2
updates:
# GitHub actions dependencies
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
# Python dependencies
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
```

Reply:
@kuisathaverat from what I can tell, dependabot doesn't support updating the pre-commit hooks themselves: https://github.com/dependabot/dependabot-core/issues/1524